app/assets/stylesheets/_inset-text.scss

```diff
@@ -32,3 +32,7 @@
     color: $govuk-error-colour;
   }
 }
+
+.app-inset-prompt {
+  background-color: #f3f2f1;
```

Comment:
is there some sort of sass variable for this colour we could use or is it a different colour not in any of the govuk frontend stuff? cc @simonwhatley

Reply:
Done